test/e2e/nsx_vm_test.go

```diff
@@ -0,0 +1,75 @@
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateVM(t *testing.T) {
+	t.Run("testCreateVMBasic", func(t *testing.T) { testCreateVMBasic(t) })
+}
+
+func testCreateVMBasic(t *testing.T) {
+	_, deadlineCancel := context.WithTimeout(context.Background(), defaultTimeout)
+	defer deadlineCancel()
+
+	ns := "test-create-vm-basic-2"
+
+	err := testData.createVCNamespace(ns)
+	if err != nil {
+		t.Fatalf("Failed to create VC namespace: %v", err)
+	}
+	defer func() {
+		err := testData.deleteVCNamespace(ns)
+		if err != nil {
+			t.Fatalf("Failed to delete VC namespace: %v", err)
+		}
+	}()
+
+	// Create public vm
+	storageClassName, storagePolicyID, _ := testData.vcClient.getStoragePolicyID()
+	// vc ui uses - instead of _
```

Comment:
Should it be
vc ui uses _ instead of -
or
storageclass name uses - instead of _?

Reply:
OK